package.json

```diff
@@ -8,6 +8,7 @@
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
     "@microlink/react-json-view": "^1.22.2",
+    "@monaco-editor/react": "^4.6.0",
```

Comment:
Please also remove `prismjs` as there don't appear to be any other usages of it.

Reply:
Update: it seems like the monaco-editor/react package is pretty challenging to unit test, and I felt uncomfortable replacing all instances of `WDLViewer` with this package only to have pre-existing unit tests break without a clean way to fix them. The rest of the team also felt very strongly about using this editor, so in an attempt to not scrap it completely we've opted for the current solution:

- Keep the existing `WDLViewer`. Any updates to its uses have been reverted.
- import `monaco-editor/react` package and use in a new `WDLEditor` component.
- For now, the `WDLEditor` will be contained within the `workflows` folder and only used by one component, `WorkflowModal`.
- Although I'm unable at this time (for the sake of the migration) to get the `WDLEditor` to render any wdl content, I did create a unit test that checks for the styling on the Editor element to verify that it does load onto the page.